script/core/monitors/MarketHealth.s.sol

```diff
@@ -0,0 +1,156 @@
+// SPDX-License-Identifier:	AGPL-3.0
+pragma solidity ^0.8.13;
+
+import {Deployer} from "mgv_script/lib/Deployer.sol";
+import {Test2, console2 as console} from "mgv_lib/Test2.sol";
+import {UpdateMarket} from "mgv_script/periphery/UpdateMarket.s.sol";
+import {MgvReader, VolumeData, IMangrove} from "mgv_src/periphery/MgvReader.sol";
+import {IERC20} from "mgv_src/IERC20.sol";
+import {MgvStructs} from "mgv_src/MgvLib.sol";
+
+/**
+ * @notice Script to obtain data about a given mangrove half book. Data is outputted to terminal as space separated values.
```

Comment:
The doc could use a small example of what the script outputs -- at the moment I have trouble having a clear picture.